### PR TITLE
Examples usages of GridModel.ensureSelectionVisible()

### PR DESCRIPTION
**Pull request by peterjcoffman**
Adds a scroll button to the existing Grid Performance test page.

Also renames the Grid Performance tab to just Grid, since it relates to more than just performance.

### CODE REVIEW

File: client-app/src/desktop/tabs/inputs/tabs/TimeInputPanel.js [hunk 1]
```diff
@@ -0,0 +1,70 @@
+import {useLocalModel} from '@xh/hoist/core/hooks';
+import {InputTestModel} from '../InputTestModel';
+import {PropTypes as T} from 'react-view';
+import {TimeInput} from '@xh/hoist/desktop/cmp/input';
+import {Icon} from '@xh/hoist/icon';
+import {hoistCmp} from '@xh/hoist/core';
+import {inputTestPanel} from '../InputTestPanel';
+
+export const TimeInputPanel = hoistCmp({
```
Comment (amcclain) [inline client-app/src/desktop/tabs/inputs/tabs/TimeInputPanel.js]: This file does not look like it should be included in this pull request. Was this branch created directly off of develop?

**Comment by lbwexler** [comment]
On the first point -- I discussed this with Peter.  We planned to abandon
that PR, and he was preparing another branch that places a test of the new
functionality in the admin grid testing area.

-Lee

**Comment by peterjcoffman** [comment]
> How about placing the button in the "Display Options" fly-out for the sample grids? That is already clearly set aside as a Toolbox-specific set of example controls, different from anything you would have in an app. I think we can even keep the title of that panel the same - scroll-to-selected is closely enough related to "display" IMO to leave that title in place.

That's not entirely straightforward. The 'Display Options' fly-out expects an AgGridModel, not a GridModel. We use that fly-out in places where no GridModel even exists, such as the ag-Grid Wrapper.

**Comment by peterjcoffman** [comment]
>let's have a quick discussion first to see if we want to do anything with a deeper integration here, or just call the method in a one-off reaction within the Portfolio example.

It would be really trivial to add this behavior to the TreeMapModel. Just add 
```        this.gridModel.ensureSelectionVisible();```
to the bottom of the defaultOnClick method.

**Comment by lbwexler** [comment]
Let's just remove it from the public area, so we can merge the button in the test area and move on.  Would much rather we work on adding this behavior to TreeMapModel now.

**Comment by lbwexler** [comment]
please create a new ticket in hoist-react for the treeMap work and get started on it.
(Think we will want to do it on syncSelection() rather than the click handler, but will let you take a look)

**Pull request closed without merging**